app-text/alanvar/alanvar-0.5.0.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Al-Anvar, a Quran Study Software "
+DESCRIPTION="Al-Anvar is a cross-platform Quran Study Software. It uses \
+\"Tanzil.net\" and \"Zekr.org\" database as its database."
+
+HOMEPAGE="http://al-anvar.sourceforge.net/"
+COPYRIGHT="2012-2018 S. Hosein Hoseini"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mazbrili/al-anvar/archive/$portVersion.zip"
+CHECKSUM_SHA256="3f3e57d453302fe11b715e75bce0b2c630cae70143987cf4208c5d8285a52817"
+SOURCE_DIR="al-anvar-$portVersion"
+ADDITIONAL_FILES="alanvar.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	alanvar$secondaryArchSuffix = $portVersion
+	app:Al_anvar = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+  	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+  	lib:libphonon4qt5$secondaryArchSuffix
+  	lib:libQtSql$secondaryArchSuffix
+  	lib:libsqlite3$secondaryArchSuffix
+  	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+ 	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+  	devel:libQtSql$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+  	devel:libQt5Xml$secondaryArchSuffix
+  	devel:libz$secondaryArchSuffix
+	devel:libphonon4qt5$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:unzip
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -f ../bin/al-anvar $appsDir/Al-anvar
```

Comment:
Suggestion:
```diff
-	mkdir -p $appsDir
+	install -d "$appsDir"
-	cp -f ../bin/al-anvar $appsDir/Al-anvar
+	install -T ../bin/al-anvar "$appsDir"/Al-anvar
```
Hmm, are you sure about the `../bin/` ?
(When haikuporter calls `INSTALL()` (and `BUILD()`) the current directory is the first directory of the tarball.)

BTW, if it makes sense to rename the runtime to `Al-Anvar` (instead of `Al-anvar`) then please feel free to do so (and adjust the `PROVIDES` accordingly.)